VCProject.Base/VCHelper.cs

```diff
@@ -41,6 +41,12 @@ private VCFileConfiguration GetVCFileConfig(Document document, out string reason
                 return null;
             }
 
+            if (vcFile.FileType == eFileType.eFileTypeCppHeader)
+            {
+                reasonForFailure = ""; // Empty string == skip output for it.
+                return null;
+            }
+
             if (vcFile.FileType != eFileType.eFileTypeCppCode && vcFile.FileType == eFileType.eFileTypeCppClass)
```

Comment:
Without the check for it being a eFileType.eFileTypeCppHeader it got stuck trying to *compile* header files. (i.e.  eFileTypeCppCode is true for .h files, but trying to compile those doesn't work well for me)

I'll try and make it nicer! :)